subprojects/sourcepawn/Listeners.cpp

```diff
@@ -266,4 +266,21 @@ namespace SPExt::Listener
         return true;
     }
 
+    SPMod::IForward::ReturnValue MessageHookCallback(SPMod::IMessage *const message, std::any cbData)
+    {
+        auto func = std::any_cast<SourcePawn::IPluginFunction *>(cbData);
+        if (func && func->IsRunnable())
+        {
+            // typedef MessageHandler = function PluginReturn (MessageDest dest, int type, int receiver);
+            cell_t retValue = 0;
```

Comment:
Is it being read in `Execute`? So do we need to initialize it to `0`?

Reply:
I guess it's unnecessary initializing it, but it's not harmful either.

Reply:
It's harmful as it tends you to think that it matters to initialize it with `0`.